### PR TITLE
[feat] Added mobile styles for alerts

### PR DESCRIPTION
**Pull request by y-lohse**
Update the alerter style to work on mobile.

The `coz-overlay` class is removed because it is never actually used.

Just a head's up, when this PR is merged I'm going to need a new published version of cozy-ui :-)

### CODE REVIEW

File: stylus/ui-components/alerts.styl [hunk 1]
```diff
@@ -10,39 +10,20 @@ $alerts
...
+        bottom           50px
         left             0
-        margin           0 em(16px)
-        border-radius    em(10px)
+        right            0
         background-color emerald
         box-shadow       0 em(6px) em(18px) 0 rgba(50, 54, 63, .23)
         box-sizing       border-box
-        padding          1em 1.5em
+        padded(.8em, 1em)
```
Comment (GoOz) [inline stylus/ui-components/alerts.styl]: Just wondering, does it work if you write `padded .8em, 1em` or not?

File: stylus/ui-components/alerts.styl [hunk 2]
```diff
@@ -69,7 +50,14 @@ $alerts
...
-            left        50%
-            max-width   alert-width
-            margin-left -(alert-width/2)
-            width       100%
+            top              em(16px)
+            bottom           auto
+            left             50%
+            margin-left      -(alert-width / 2)
+            max-width        alert-width
+            padded(1em, 1.5em)
```
Comment (GoOz) [inline stylus/ui-components/alerts.styl]: Same here.

File: stylus/ui-components/alerts.styl (hunk 1 above)
Comment (y-lohse) [inline stylus/ui-components/alerts.styl]: Yes it does! I keep forgetting that, amending the PR.

**Comment by y-lohse** [comment]
@GoOz  Ok, all good now; see the latest addition to z-indexes, just in case.

**Comment by GoOz** [comment]
Man, Z-Index is like a plague 😆 
That'll do it for now. Thanks. 🎉

**Pull request merged**